VMEncryption/main/handle.py

```diff
@@ -1552,7 +1559,7 @@ def daemon_encrypt():
         except Exception as e:
             message = "Failed to encrypt data volumes with error: {0}, stack trace: {1}".format(e, traceback.format_exc())
             logger.log(msg=message, level=CommonVariables.ErrorLevel)
-            hutil.do_exit(exit_code=0,
+            hutil.do_exit(exit_code=CommonVariables.encryption_failed,
```

Comment:
What will changing the exit codes do here? Help in filtering or something else?

Reply:
Yes, this helps to filter the telemetry for backend QOS reporting, as it allows categorizing failure types by exit code.